learning-guide/beginner.md

```diff
@@ -16,7 +16,7 @@ Click the image to view on YouTube.
 | Title | Content | Length | 
 | --- | :-- | --- |
 | The Complete Guide to Full Stack Ethereum Development - Tutorial for Beginners [2021] | [![video](https://img.youtube.com/vi/a0osIaAOFSE/0.jpg)](https://www.youtube.com/watch?v=a0osIaAOFSE) | 1h 01m 04s
-| Local Solidity Development | [View](https://hardhat.org/) |
+| Local Solidity Development | [![video](https://user-images.githubusercontent.com/68073132/136015000-4801b909-f183-4726-b2c5-e2ce82aa2d31.png)](https://hardhat.org/) |
```

Comment:
Could you write in length = Written content? Sample below

| Title | Content | Length | 
| --- | :-- | --- |
| Local Solidity Development | [![video](https://user-images.githubusercontent.com/68073132/136015000-4801b909-f183-4726-b2c5-e2ce82aa2d31.png)](https://hardhat.org/) | Written Content |

Reply:
Sure